node/narwhal/tests/common/primary.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use itertools::Itertools;
+use snarkos_account::Account;
+use snarkos_node_narwhal::{
+    helpers::{init_primary_channels, Committee, PrimarySender, Storage},
+    Primary,
+    MAX_GC_ROUNDS,
+};
+
+use tracing::*;
+
+use indexmap::IndexMap;
+use parking_lot::RwLock;
+use rand::SeedableRng;
+use std::{collections::HashMap, sync::Arc};
+
+use crate::common::CurrentNetwork;
+
+// Initializes a new test committee.
+pub fn new_test_committee(n: u16) -> (Vec<Account<CurrentNetwork>>, Committee<CurrentNetwork>) {
+    const INITIAL_STAKE: u64 = 1000;
+
+    let mut accounts = Vec::with_capacity(n as usize);
+    let mut members = IndexMap::with_capacity(n as usize);
+    for i in 0..n {
+        // Sample the account.
+        let account = Account::new(&mut rand_chacha::ChaChaRng::seed_from_u64(i as u64)).unwrap();
+
+        // TODO(nkls): use tracing instead.
+        println!("  Validator {}: {}", i, account.address());
```

Comment:
looks like `tracing` is already imported and used, any reason why it's not used here yet?